objcgen/objcgenerator.cs

```diff
@@ -692,7 +653,6 @@ void GenerateDefaultValuesWrapper (string name, MethodBase mb, ParameterInfo[] p
 			headers.WriteLine ();
 
 			builder.BeginImplementation ();
-			implementation.Write ('\t');
```

Comment:
I guess since this wasn't replaced by any additional call it is no longer needed